main.go

```diff
@@ -97,6 +105,8 @@ func main() {
 		"use the demo provisioner to set host states")
 	flag.StringVar(&healthAddr, "health-addr", ":9440",
 		"The address the health endpoint binds to.")
+	flag.IntVar(&webhookPort, "webhook-port", 0,
```

Comment:
What's the reason for defaulting it to disabled?

Reply:
I did not want to affect make run command on local. Maybe I can enable webhook by default and deliberately disable on make run?

Reply:
That sounds plausible. I wonder what other projects do...

Reply:
I checked cluster-api-provider-metal3 as an example and it also defaults to disabled.

Reply:
Hi! Just FYI, I am not sure if we want to leave it disabled in BMO, but if you are referring to [this](https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/main.go#L78) in CAPM3 which is disabled by default, for now, it will be changed soon as part of preparing for CAPI v1alpha4 changes in CAPM3. You can check this doc for [ref](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#required-kustomize-changes-to-have-a-single-manager-watching-all-namespaces-and-answer-to-webhook-calls) which includes `Changes in the main.go file`.

Reply:
thanks @furkatgofurov7, I'll change that way and handle `make run` works without webhook.

Reply:
There is a bug in kustomize to merge args parameters and if I change it to enabled by default, there will no longer need to upgrade kustomize to 3.10.0.